src/sentry/static/sentry/app/views/releasesV2/detail/releaseHeader.tsx

```diff
@@ -169,18 +154,13 @@ const StatsWrapper = styled('div')`
 `;
 
 const DeploysWrapper = styled('div')`
-  display: flex;
-  margin-top: ${space(0.5)};
-  @media (max-width: ${p => p.theme.breakpoints[0]}) {
-    flex-wrap: wrap;
-  }
+  white-space: nowrap;
 `;
 
-const StyledBadge = styled(Tag)`
-  background-color: ${p => p.theme.gray4};
-  font-size: ${p => p.theme.fontSizeSmall};
-  color: ${p => p.theme.white};
-  margin-left: ${space(0.5)};
+const StyledDeployBadge = styled(DeployBadge)`
+  margin-right: 0;
+  margin-left: ${space(1)};
```

Comment:
So I generally prefer to have wrappers to control spacing otherwise there's a constant dance between overriding margins. @robinrendle @doralchan any thoughts/guidelines here?

Reply:
That's fair, I changed it